File: LeetCode Solutions/Problems/1936. Add Minimum Number of Rungs.cpp
class Solution {
public:
    int addRungs(vector<int>& rungs, int dist) {
        int previous = 0, minimumNumberOfRungs = 0;
        for(int r:rungs){
            minimumNumberOfRungs += (r-previous-1) / dist;
            previous = r;
        }
        return minimumNumberOfRungs;
    }
};

File: LeetCode Solutions/Problems/1936. Add Minimum Number of Rungs.py
class Solution:
    def addRungs(self, rungs: List[int], dist: int) -> int:
        previous, minimumNumberOfRungs = 0, 0
        for r in rungs:
            minimumNumberOfRungs += (r-previous-1) // dist
            previous = r
        return minimumNumberOfRungs